tests/models/idefics/test_modeling_idefics.py

```diff
@@ -71,6 +71,7 @@ def __init__(
         type_vocab_size=16,
         type_sequence_label_size=2,
         initializer_range=0.02,
+        alpha_initializer="ones",
```

Comment:
The initializer is at zero, because during training, you want the model to progressively pay attention through the training, the alphas' value grows through the training. 
However, when creating a new model to make tests, you want this value to be non-zero, otherwise, the checks for image input are abit worthless, since the alphas will zero-out anything coming out from the cross-attention

Reply:
Cool, thanks for explaining!